### PR TITLE
fix: column sort order issues

### PR DESCRIPTION
**Pull request by cbt1**
Fixed isses related to user dragging a dimension column to a different index:
- Single select did not work
- Multicell select did not work
- Totals value did not match column

### CODE REVIEW

File: src/table/utils/selections-utils.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ export const getMultiSelectedRows = (
     newSelectedRows[cell.qElemNumber] = cell.rowIdx;
     // also add the next or previous cell to selectedRows, based on which arrow is pressed
     const idxShift = evt.key === KeyCodes.DOWN ? 1 : -1;
-    const nextCell = pageRows[cell.pageRowIdx + idxShift][`col-${cell.colIdx}`] as Cell;
+    const nextCell = pageRows[cell.pageRowIdx + idxShift][`col-${cell.pageColIdx}`] as Cell;
```
Comment (haxxmaxx) [inline src/table/utils/selections-utils.ts:174]: have you tried this with pagination?

Comment (cbt1) [reply src/table/utils/selections-utils.ts:174]: Yes see my other comment :) Not all that much but did test with a moved column, a non-moved column and using keyboard multiselect

**Review by a-m-dev — APPROVED** [review]
over all looks good, but not sure about which path should take on `colIdx` or `pageColIdx`.

**Review by haxxmaxx — APPROVED** [review]
I tested it and I can't break it. It was actually strange that it worked before. We set column.id to col-${pageColIdx} so this probably fixes some undiscovered bug 👍

**Pull request merged**